### ISSUE TITLE
Can't deploy @org packages

### ISSUE THREAD

**Issue by mwcz** [nodeshift/nodeshift#538]
Deploying a package with an @org-scoped name like `@myorg/mypackage` fails currently with:

```
ERROR "name" in package.json can only consist lower-case letters, numbers, and dashes. It must start with a letter and can't end with a -.
```

Since these are valid npm package names, it would be nice if they could be supported.

I assume that allowing `@org/pkg` names naively would interfere with nodeshift's automatic name generation for resources within openshift.  Maybe a transform like this would suffice?

```
@org/pkg -> org__pkg
```

**Comment by lholmquist** [comment]
@mwcz nice catch.  yea,  we should take scopes into account for the validation.   we could test out how "@org/pkg" works and if it does cause issues,  then your suggestion of "org_pkg" should work fine

**Comment by lholmquist** [comment]
@mwcz I just published 8.0.1 which should fix this

**Comment by mwcz** [comment]
Awesome, thanks Luke.
